pkg/volume/vsphere_volume/attacher.go

```diff
@@ -80,7 +80,7 @@ func (attacher *vsphereVMDKAttacher) Attach(spec *volume.Spec, nodeName types.No
 		return "", err
 	}
 
-	klog.V(4).Infof("vSphere: Attach disk called for node %s", nodeName)
+	klog.V(4).InfoS("vSphere: Attach disk called for node", "nodeName", nodeName)
```

Comment:
```suggestion
	klog.V(4).InfoS("VSphere: Attach disk called for node", "nodeName", nodeName)
```

Reply:
Thanks for your suggestion, but vSphere probably is the custom abbreviation.